plugins/optimization-detective/tests/test-optimization.php

```diff
@@ -338,21 +339,21 @@ public function data_provider_test_od_can_optimize_response(): array {
 	}
 
 	/**
-	 * Test od_can_optimize_response().
+	 * Test od_get_cannot_optimize_reasons().
 	 *
-	 * @covers ::od_can_optimize_response
+	 * @covers ::od_get_cannot_optimize_reasons
```

Comment:
This test could then test both `od_get_cannot_optimize_reasons()` and `od_can_optimize_response()`